src/content/snapshot/branching-and-baselines.md

```diff
@@ -137,30 +153,56 @@ When you create a new build for a new commit, Chromatic will calculate a baselin
 
 The ancestor build is the most recent ancestor (commit) in the git history that has had Chromatic run against it. Often, it is the previous commit:
 
-```shell
+<!-- ```shell
 x - Build N
 |
 y - Build N+1
+``` -->
+
+```mermaid
+
+gitGraph
+ commit id: "x (Build N)"
+ commit id: "y (Build N+1)"
 ```
 
 If you don’t run CI on every commit (which is common if you don’t push every single time you commit), there may be a gap:
 
-```shell
+<!-- ```shell
 x - Build N
 |
 y
 |
 z - Build N+1
+``` -->
+
+```mermaid
```

Comment:
It's just a small thing; nothing is blocking it, but could we change the font size a bit for readability or make it bolder? If not, that's ok. Just wondering

Reply:
@jonniebigodes there's a page detailing [customizing gitgraph diagrams](https://mermaid.js.org/syntax/gitgraph.html), but I'll warn you that I've had some issues scrolling through that page specifically in the past (the info is there, it just might start scrolling you somewhere random lol).  You should be able to make adjustments to font size and such through `themeVariables`, not sure if y'all are looking to standardize it through `astro.config.mjs`, but I was throwing something similar to the below in my charts to help with readability:

```
%%{init: { 'theme': 'dark' , 'themeVariables': {
					'commitLabelFontSize': '12px',
					'commitLabelBackground': '#686868'
					} } }%%
```

@winkerVSbecks quick question: do you have any guidance around whether we should add any customization on-the-fly or would you prefer we keep to the standardized theme set in the astro config?  Aside from the general readability things mentioned by João, I'm assuming there may be times when a light amount of customization would be beneficial depending on what's trying to be conveyed (ex: horizontal vs. vertical charts).  I'm completely comfortable discussing with y'all first if these pop up, but I wanted to check what your general thoughts around this are and what your preference would be in approaching when/if we hit these situations.

Reply:
Yea, we can totally customize inline. I did a bunch of that in this PR too. That said, I'll add the themeVariables to increase the base font-size.

Reply:
@repomadness I remember that page and the scrolling issues. But on a quick glance now, it seems that they solved them. @winkerVSbecks appreciate it 🙏